src/library/scala/collection/immutable/LazyList.scala

```diff
@@ -253,7 +274,9 @@ final class LazyList[+A] private(private[this] var lazyState: () => LazyList.Sta
     // if it's already mid-evaluation, we're stuck in an infinite
     // self-referential loop (also it's empty)
     if (midEvaluation) {
-      throw new RuntimeException("self-referential LazyList or a derivation thereof has no more elements")
+      throw new RuntimeException(
+        "LazyList evaluation depends on its own result (self-reference); see docs for more info"
+        )
```

Comment:
I recently inherited a more modern monitor, so this looks short enough for one line. But I did not count chars.

Reply:
after my change, part of the last character and closing quote run off the end of the window. on the other hand, I use an unusually large font size. on the other other hand, we should try to accommodate as many people as possible, so erring shorter is maybe better anyway?

in its current state in the PR, the line has 97 characters. if I put the message on the same line as the constructor, it has 123